### ISSUE TITLE
UE4Editor-XMPP.pdb is not loaded while debuging in VS2017

### ISSUE THREAD

**Issue by super-NOV-a** [harvard-edge/airlearning-ue4#14]
I met a problem while building the UE project. Here is my environment:
Win11, Visual Studio 2017, UE 4.18.3, 
I successfully generated the AirLearning.sln from AirLearning.uproject, but when I debug AirLearning project in Visual Studio using DebugGame Editor, it throws out this in **CHINESE**:
未加载UE4Editor-XMPP.pdb   (UE4Editor-XMPP.pdb is not loaded)
0x00007FF83C414FA3 (UE4Editor-XMPP.dll)处(位于 UE4Editor.exe 中)引发的异常: 0xC0000005: 读取位置 0xFFFFFFFFFFFFFFFF 时发生访问冲突。  (Something wong about UE4Editor-XMPP.dll happened, it is about ERROR: 0xC0000005: reading location 0xFFFFFFFFFFFFFFFF visiting crash)
![image](https://github.com/harvard-edge/airlearning-ue4/assets/70046229/c92e296c-51cb-4820-b6b0-b5b4c74c5ec3)
It seems like I cannot find the .pdb file of the project.
Anyone could give me any advice? Thanks for all of you guys.

**Comment by super-NOV-a** [comment]
I tried to switch the UE4Editor-XMPP.dll file in the given path, I copied the file from version 4.27. This way, I can debug the .sln file and open UE4 environment, but after a short period, it got a new problem. And I also  trid to change the version of the project, but it failed to debug. So I deleted the .sln file and wanted to generate a new one, but I found that it just cannot generate a new .sln file (I guess it is because of the logs and some already generated files), so I tried to hit LAUNCH GAME button, when I clicked, the game environment just worked. Although there is no other buttons in the env, I can run airlearning-RL codes now.
![image](https://github.com/harvard-edge/airlearning-ue4/assets/70046229/f0753ac7-85de-42cf-8202-d016d59c4e61)
